packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFNavigationDelegateHostApi.m

```diff
@@ -152,6 +170,22 @@ - (void)webView:(WKWebView *)webView
                                       }];
 }
 
+- (void)webView:(WKWebView *)webView
+    decidePolicyForNavigationResponse:(WKNavigationResponse *)navigationResponse
+                      decisionHandler:(void (^)(WKNavigationResponsePolicy))decisionHandler {
+  [self.navigationDelegateAPI
+      decidePolicyForNavigationResponseForDelegate:self
+                                           webView:webView
+                                navigationResponse:navigationResponse
+                                        completion:^(FWFWKNavigationResponsePolicyEnumData *policy,
+                                                     FlutterError *error) {
+                                          NSAssert(!error, @"%@", error);
+                                          decisionHandler(
+                                              FWFNativeWKNavigationResponsePolicyFromEnumData(
```

Comment:
This has the some problem as the code discussed recently in the permission callback PR; it will unconditionally call the callback with an invalid value if Apple ever adds a new enum value. @bparrishMines Are you going to mass-fix this in a fast follow? I'm concerned about replicating a pattern that we know is dangerous.

Reply:
This is fixed now with https://github.com/flutter/packages/pull/3812. 

This also only affects enums that we receive from Objective-C. The Dart code shouldn't be able to return an enum the native side doesn't recognize.